jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -614,6 +609,81 @@ def vue_update_subset(self, *args):
 
         self._push_update_to_ui()
 
+    def update_subset(self, subset_label, subregion=None, **kwargs):
```

Comment:
can we have `subset_label` be optional as well and grab from the dropdown if not passed?

Reply:
Done.

Reply:
The following code is giving me a traceback that the `subset_label` is required and missing, even though I have Subset 1 selected in the drop down (at the top of the app). I interpreted @kecnry as saying that should assume that Subset 1 is to be updated. Was that incorrect?
```
sp = cubeviz.plugins['Subset Tools']
sp.update_subset()

TypeError: SubsetTools.update_subset() missing 1 required positional argument: 'subset_label'
```

Reply:
Whoops, VSCode didn't want me to save the file and then I missed making it an optional arg when I reverted and did it again. Should be fixed now.